prime/src/components/borrow/actions/AloeAddMarginActionCard.tsx

```diff
@@ -80,7 +81,12 @@ export function AloeAddMarginActionCard(prop: ActionCardProps) {
         options={dropdownOptions}
         selectedOption={selectedTokenOption}
         maxAmount={maxString}
-        onChange={(value) => callbackWithFullResult(selectedToken, value)}
+        onChange={(value) => {
+          const output = formatNumberInput(value);
+          if (output != null) {
+            callbackWithFullResult(selectedToken, value);
+          }
+        }}
```

Comment:
Seems like all of these should be done inside `callbackWithFullResult`. Is there a reason you did it down here?